app/thirdparty/java/other/writeily/model/WrMarkorSingleton.java

```diff
@@ -133,6 +137,14 @@ public void moveSelectedNotes(List<File> files, String destination, final Contex
         }
     }
 
+    public void copyContentOfFile(File file,Context context) {
```

Comment:
This functionality is duplicate as well. Juse use ` markor/app/src/main/java/net/gsantner/opoc/util/ShareUtil.java ::setClipboard(text)`

Reply:
Fix in wladokacur/markor@6c6f39893eb4315466faf99062318f8da95c45a8